agents/kathy/Cargo.toml

```diff
@@ -22,7 +22,9 @@ tracing-futures = "0.2.4"
 tracing-subscriber = "0.2.15"
 rand = "0.8.3"
 prometheus = "0.12"
+nomad-xyz-configuration = { git = "https://github.com/nomad-xyz/configuration", rev = "68d2403" }
 
-nomad-base = {path = "../../nomad-base"}
-nomad-core = {path = "../../nomad-core"}
+nomad-types = {git = "https://github.com/nomad-xyz/rust", rev = "d4d02aa"}
```

Comment:
isn't this in the same workspace? o.O

Reply:
Haha yeah I ran in circles trying to get consistent versioning between `rust` and `configuration` so not sure this was necessary or not. Either way doesn't matter since I'll need to manually make PR to bump to point to latest `main` once both config and rust PRs are in

Reply:
Yeah so all part of the versioning issues I had. I do indeed need to use commit hashes instead of paths or branches but relative paths/branches will be fixed after merge